### DISCUSSION TITLE
Deno deploy ISOLATE_INTERNAL_FAILURE

### DISCUSSION THREAD

**Discussion by Pamel-a** [denoland/deno#21834] [Q&A]
I'm trying to learn how to use deno, i'm trying to deploy this [MongoDB][1] project

But on the Deno deploy website I'm getting this error:
```js
Cloning repository (100%)
Project linked successfully. Building first deployment:
Downloaded
file:///src/config.js
Packaging complete
Error The deployment failed: ISOLATE_INTERNAL_FAILURE
```
I could not find much info about this error, I wonder if i'm doing something wrong in the imports or if it doesn't support koa or mongo?

  [1]: https://github.com/Pamel-a/mongo

**Comment by huntercaron** [comment]
In my case I got this when exporting the wrong thing from my deploy function. May be related to that?

My situation was fixed by changing my hono export from `export default app.fetch` to `export default app`
